src/styles/_App.scss

```diff
@@ -244,3 +244,18 @@
 hr {
   margin: 75px 0;
 }
+
+.section-header div:nth-child(2) {
+  align-self: flex-end;
+}
+
+@media (min-width: 768px) {
+  .section-button .text-right * {
+    margin-left: 3px;
+  }
+}
+@media (max-width: 767px) {
```

Comment:
When dealing with the same rule across widths, margin in this case, I suggest we define a base style and then override with either a min-width or a max-width instead of defining max and min around the same breakpoint.

for example

```css
.section-button .text-right * {
  margin: 3px 0 0 0;
}

@media (min-width: $sm) {
  .section-button .text-right * {
    margin: 0 0 0 3px;
  }
}
```